src/components/CommentList.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { ErrorType } from '../types/ErrorType';
+import { customFetch } from '../utils/fetchClient';
+import { Comment } from '../types/Comment';
+import { CommentInfo } from './CommentInfo';
+
+type Props = {
+  comments: Comment[];
+  setErrorType: React.Dispatch<React.SetStateAction<ErrorType | null>>;
+  setComments: React.Dispatch<React.SetStateAction<Comment[]>>;
+};
+
+export const CommentsList: React.FC<Props> = ({
+  comments,
+  setComments,
+  setErrorType,
+}) => {
+  const handleDeleteComment = async (commentId: number) => {
+    try {
+      setComments(prevComments => prevComments
+        .filter(({ id }) => id !== commentId));
+      await customFetch.removeComment(commentId);
```

Comment:
Error handling is not descriptive enough. It just sets the error type to 'DELETE', but doesn't handle the error object returned from the failed API call. It should provide more detailed message about what went wrong.